packages/entities/entities-routes/src/components/RouteList.vue

```diff
@@ -349,7 +349,7 @@ const resetPagination = (): void => {
 /**
  * loading, Error, Empty state
  */
-const errorMessage = ref('')
+const errorMessage = ref<{ title?: string, message?: string } | null>(null)
```

Comment:
If you're using this interface everywhere, instead of copy/paste, can we export the type from the `entities-shared` package and use it as needed instead of replicating?